insonmnia/hub/metrics.go

```diff
@@ -0,0 +1,19 @@
+package hub
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	tasksGauge = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Why gauges, not counters?

Reply:
Counters have only Inc() and no Dec() : )

Reply:
Oh, okay.